### PR TITLE
[NEO] Implementing Compleated mechanic and hybrid phyrexian mana (ready for review)

### PR DESCRIPTION
**Pull request by theelk801**
To do:
- [x] Implement Tamiyo, Compleated Sage
- [x] Implement hybrid phyrexian mana
- [x] Implement Compleated mechanic
- [x] Write tests

### CODE REVIEW

**Comment by theelk801** [comment]
looks like everything is passing besides the [[Helm of the Host]] test, hopefully I can figure that out tomorrow

File: Mage.Tests/src/test/java/org/mage/test/cards/copy/SparkDoubleTest.java [hunk 1]
```diff
@@ -68,7 +68,6 @@ public void test_CopyPlaneswalkerWithoutLegendaryWithOneCounter() {
         castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Spark Double");
         setChoice(playerA, true);
         setChoice(playerA, "Ajani, the Greathearted");
-        setChoice(playerA, "Ajani, the Greathearted"); // two etb effects (own + copy)
```
Comment (JayDi85) [inline Mage.Tests/src/test/java/org/mage/test/cards/copy/SparkDoubleTest.java:71]: Must be two effects for choose. Why it disappear? User cant choose a starting loyalty effect now?

Comment (theelk801) [reply Mage.Tests/src/test/java/org/mage/test/cards/copy/SparkDoubleTest.java:71]: It's not applying the way it used to, even though it's technically still an etb effect. It shouldn't matter as it's only adding counters.

**Comment by theelk801** [comment]
That's in there, I updated scryfall and gatherer downloads although the gatherer ones aren't available yet.

**Pull request merged**